src/sql/src/plan/query.rs

```diff
@@ -510,14 +510,19 @@ pub fn plan_copy_from_rows(
     Ok(expr.map(map_exprs).project(project_key))
 }
 
-/// Common information used for DELETE and UPDATE plans.
+/// Common information used for DELETE, UPDATE, and INSERT INTO ... SELECT plans.
 pub struct ReadThenWritePlan {
     pub id: GlobalId,
-    /// WHERE filter.
+    /// Read portion of query.
```

Comment:
To me, "read portion" is less clear than "WHERE filter". I like the NOTE below (which implies that even of selection is some, uh empty expr (what even is it for no WHERE?), that we still do a read.

Reply:
> what even is it for no WHERE?

For example if you have a query like `DELETE FROM t;` (where `t` is some table), then `selection` will probably be some `HirRelationExpr::Select` that reads every row of `t`, which we then use to retract every row.